service/user.service.js

```diff
@@ -4,14 +4,22 @@ const tokenService = require("../service/token.service");
 const UserDTO = require("../dto/user.dto");
 const ApiError = require("../helper/api.error");
 
-const signUp = async (name, email, password) => {
+const signUp = async (first_name, last_name, phone_number, email, password) => {
     const candidate = await UserModel.findOne({ email });
 
     if (candidate) throw ApiError.BadRequest("User already exist");
 
     const hashPass = bcrypt.hashSync(password, 10);
-    const user = await UserModel.create({ name, email, password: hashPass });
-    const userDto = new UserDTO(user);     // name, email, id
+    const user = await UserModel.create({
+        first_name,
+        last_name,
+        phone_number,
+        email,
+        password: hashPass,
+        registered_at: new Date(),
```

Comment:
@ikadushkina to not bother oneself by `new Date()` creation you can delegate it to mongoose using [timestamps](https://mongoosejs.com/docs/timestamps.html) schema feature. Just define an appropriate config in your schema, smth. like `  timestamps: {
    createdAt: 'registered_at',
    updatedAt: false
  }`